scripts/variantstore/wdl/GvsTieoutPgenToVcf.wdl

```diff
@@ -159,6 +159,8 @@ task Tieout {
     >>>
     output {
         File? tarball = "tieout.tgz"
+        Array[File] vcfs = glob("tieout/vcf/vcf_compare_chr*")
```

Comment:
Making these arrays of VCFs available as outputs in order to allow me to manually verify that the gts are no-called.